src/helpers/files.ts

```diff
@@ -332,3 +333,26 @@ export function removeFile(projectRoot: string, filePath: string): void {
     }
   })
 }
+export function getBlocklist() {
+  return [...manualBlocklist(), ...globBlocklist()]
+}
+
+export function cleanCoverageFilePaths(projectRoot: string, paths: string[], ignoreGlobs: string[]) {
+  UploadLogger.verbose(`Preparing to clean the following coverage paths: ${paths.toString()}`)
+  const coverageFilePaths = [... new Set(paths.filter(file => {
+    return fileExists(projectRoot, file)
+  }))]
+
+  if (coverageFilePaths.length === 0) {
+    throw new Error('Error while cleaning paths. No paths matched existing files!')  
+  }
+
+  const ignoredFiles = micromatch(coverageFilePaths, ignoreGlobs)
+
+  const filesAfterCheckingIgnore = coverageFilePaths.filter(path => {
+    return !ignoredFiles.includes(path)
```

Comment:
```suggestion
    return !ignoredFiles.has(path)
```

Reply:
`has()` does not exist on `string[]` What is the difference?

Reply:
You're correct. `has()` exists on instances of the Set class. It has a sublinear runtime complexity but, since `ignoredFiles` is an array, it can't actually be used in this case.